modules/ROOT/pages/_partials/endpoint-sftp-receive.adoc

```diff
@@ -0,0 +1,69 @@
+== SFTP Receive Settings
+
```

Comment:
We should have a brief explanation of what these are (probably all the reference topics should have them). Where are they used, or what part of the flow (partner > me? me > partner?)

Reply:
Good point. To be added.